### DISCUSSION TITLE
mail sending problem

### DISCUSSION THREAD

**Discussion by pmatuszy** [healthchecks/healthchecks#652] [General]
Hi
I have configured system and it looks great. Integration with email works ... sort of... I can put the address, test it (email is sent) but when the probe goes up or down I receive nothing.... Have you seen such a behaviour before?
Regards
Paul

**Comment by cuu508** [comment]
Make sure `manage.py sendalerts` is running – https://github.com/healthchecks/healthchecks#sending-status-notifications

**Reply by pmatuszy** [reply]
you made my day! I just started this and got gazillion of emails!
Thank you
Paul

**Comment by pmatuszy** [comment]
one more question: I run gunicorn in non-debug mode  - sending notinfications should be done anyway with manage.py script?

**Reply by cuu508** [reply]
Yes. The webserver and the notification sender are separate processes, and `manage.py sendalerts` is the correct way to run the notification sender.
